packages/shared/web/src/header/borrow-dropdown/borrow-dropdown.tsx

```diff
@@ -29,6 +30,7 @@ export function BorrowDropDown() {
     <DropdownButton
       popupId="borrow"
       labelKey={messages[NAV_DROPDOWN.BORROW]}
+      onClick={() => history.push('/borrow-variable')}
```

Comment:
Is it possible to wrap this up in a Link component rather than use a push? It should show up better in the DOM